middleware/idempotency/storage.go

```diff
@@ -0,0 +1,87 @@
+package idempotency
+
+import (
+	"sync"
+	"time"
+)
+
+type Storage interface {
```

Comment:
Why did you create new storage interface. I think you can make this like csrf, cache middlewares

Reply:
We don't need the following `fiber.Storage` methods:

- `Delete(key string) error`
- `Reset() error`
- `Close() error`

All types satisfying the `idempotency.Storage` interface also implement the `fiber.Storage` interface.

Reply:
Storage is generic interface for various fiber middlewares. I think we shouldnt create new interface. I can't see any desadvantage of using fiber.Storage. We should make code as simple as possible and we should avoid much copying i think.

Reply:
Would suggest the same here, and maybe also document it as something like:

> Those methods are unused so simply put a `panic("unexpected call")` would be sufficient.

Reply:
Using a `fiber.Storage` now.